ui/pages/routes/routes.component.js

```diff
@@ -698,6 +699,13 @@ export default class Routes extends Component {
 
     let isLoadingShown = isLoading && completedOnboarding;
 
+    const isConfirmation = Boolean(
```

Comment:
Will this only ever be for transaction confirmations or all of them?

Should we be explicit and call this `isTransactionConfirmation`?

Reply:
I changed it to `isConfirmTransactionRoute`